tmpdocs/powerai3.sh

```diff
@@ -0,0 +1,78 @@
+module load gcc/7.3.1
```

Comment:
We should think of a better place to put this. Perhaps we can make a `bin/` folder in the repo root.